jb-yoda/src/test/java/life/jukebox/JBWatcherTest.java

```diff
@@ -0,0 +1,31 @@
+import static org.junit.Assert.*;
+import org.junit.*;
+import java.util.concurrent.*;
+import java.util.*;
+import java.io.*;
+
+public class JBWatcherTest {
+
+  private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+  JBWatcher watcher;
+  
+  @Before 
+  public void createWather() {
```

Comment:
rename this to setup - you're doing more than createWatcher

Reply:
fixed